Components.md

```diff
@@ -18,32 +42,111 @@
 
 - Tom Jellicoe
 
+### Repositories
+
+Code:
+
+- eodhp-ades-demonstration
+- eodhp-ades-workspace-access
+- eoepca-proc-service-template
+- stac-workflow-generator
+
```

Comment:
There are still some relevant repos in the UKEODHP org. Mainly ones that we are going to continue to use after this project, e.g. workspace-controller. Also, elsewhere we have workflow-runner and workspace and analysis system as two separate components. Here, we seem to have them grouped together (I know i'm the one who created this repo, just spotting my mistake). Should we split them, or should we keep them as one component for the sake of a holistic approach? It doesn't necessarily change who will be working on them.

- workspace-controller
- ApplicationHub
- apphub-singleuser
- ades-stagein
- ades-stageout
- calrissian
- pycalrissian (?)
- ZOO-project (?)

Probably need Tom/Alex P to confirm if the (?) repos are required.